src/deepsparse/engine.py

```diff
@@ -184,6 +185,7 @@ def __init__(
         self._input_shapes = input_shapes
         self._cpu_avx_type = AVX_TYPE
         self._cpu_vnni = VNNI
+        self._cpu_bf16 = BF16
```

Comment:
could we keep the `ARCH` object as the member and query that for the property functions?